nms/app/fbcnms-projects/magmalte/app/components/feg/FEGGatewayDialog.js

```diff
@@ -125,10 +123,27 @@ function getDiameterServerConfig(
   };
 }
 
-function getOverwriteAPN(overwrite_apn: ?string): overWriteAPN {
-  return {
-    apn: overwrite_apn || '',
-  };
+function getVirtualApnRules(rules: ?virtual_apn_rules): 
+  ?Array<[string, string]> {
+  if (!rules || rules.length == 0) {
+    return null;
+  }
+  return rules.map((pair) => {
+    return  [ pair.apn_filter, pair.apn_overwrite ];
+  });
+}
+
+function virtualApnRulesToObject(
+  props: ?Array<[string, string]>,
+): ?virtual_apn_rules {
+  if (!props) {
+    return null;
+  }
+
+  const results = props.filter(p => p[0]).map(pair => {
```

Comment:
you can remove above null check and instead do props?.filter and this should return undefined if props isn't present